docs/rpc.md

```diff
@@ -47,7 +47,7 @@ which would yield the response:
 | `getclaimable` | Yes |
 | `getconnectioncount` | Yes |
 | `getcontractstate` | Yes |
-| `getnep5balances` | No (#498) |
+| `getnep5balances` | Yes |
```

Comment:
We also have client godoc, update that please as you have implemented client's method along with the server's one.